SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -32,6 +32,7 @@
 #include "SpatialConstants.h"
 #include "SpatialGDKSettings.h"
 #include "Utils/EntityPool.h"
+#include "Utils/ErrorCodeRemapping.h"
```

Comment:
What is this used for?

Reply:
It's used for `ENetworkFailure::FromDisconnectOpStatusCode(ConnectionStatusCode)`